packages/alfa-dom/test/get-tab-sequence.spec.tsx

```diff
@@ -0,0 +1,109 @@
+import { jsx } from "@siteimprove/alfa-jsx";
+import { test } from "@siteimprove/alfa-test";
+import { getTabSequence } from "../src/get-tab-sequence";
+
+test("Computes the tab sequence of three non-prioritized elements", t => {
+  const a1 = <a href="#">Foo</a>;
+  const a2 = <a href="#">Bar</a>;
+  const input = <input type="text" value="Hello" />;
```

Comment:
I'd recommend the following fixtures as they cover both interactive and static elements:

```tsx
const div = <div />;
const a = <a href="#" />;
const input = <input type="text" />;
```

It's also a good idea to include only the bare minimum of attributes when creating fixture elements. Text (e.g. `Foo`) is sometimes useful to distinguish otherwise identical elements when debugging but can generally be left out.